packages/items_repository/lib/src/domain/items_client_i.dart

```diff
@@ -0,0 +1 @@
+abstract interface class ItemsClientI {}
```

Comment:
cose'e'?

Reply:
Sara l’api provider per andare a recuperare i dati. Il data source in poche parole